source/common/utils/logging.js

```diff
@@ -1,6 +1,78 @@
 // @flow
+import { isEmpty } from 'lodash';
+import type {
+  FormatMessageContextParams,
+  ConstructMessageBodyParams,
+  MessageBody
+} from '../types/logging.types';
+
+const DEFAULT_MESSAGE_BODY: MessageBody = {
+  at: '',
+  env: '',
+  ns: [
+    'daedalus',
+  ],
+  data: {},
+  app: [
+    'daedalus'
+  ],
+  msg: '',
+  pid: '',
+  sev: '',
+  thread: ''
+};
```

Comment:
Im not sure if we should have empty defaults for most props? Shouldn't we only define defaults that actually have a value (like `app` or `ns`) or for props that are completly optional? The way this is done now basically means that the Flow types are kind-of-useless because you won't get any errors if you forget to pass required props - and on the other hand i cannot see what is required information.

Reply:
@DominikGuzei are you OK with what @MarcusHurney changed?